README.md

```diff
@@ -11,6 +11,140 @@ Inside of the nix-shell:
 - build the documentation by running `make html`
 - view changes as you edit by running `make livehtml` and visit http://127.0.0.1:8000
 
+## Deploying
+
+### Paths
+
+In order for the version switcher to work properly, the paths where we serve the docs have to have the following structure:
+
+- Docs for latest release are at `/`
+  - Ends up being the target for old links, so old links continue to work as they always have
+- Docs for previous releases are at `/vN.N.N/`
+
+For example, if we're publishing v5.4.5, v5.4.4, and v5.4.3...
+
+- `https://docs.gravwell.io` is v5.4.5
+- `https://docs.gravwell.io/v5.4.4/` is v5.4.4
+- `https://docs.gravwell.io/v5.4.3/` is v5.4.3
+
+### Directory structure
+
+Assume we're using a web server to serve a directory as `docs.gravwell.io`.
+
+If we wanted to serve v5.4.5, v5.4.4, and v5.4.3 as shown above, we could structure the directory like this:
+
+```
+docs/     <- the full build of v5.4.5
+├─ 404.html
+├─ index.html
+├─ ...
+├─ v5.4.4   <- the full build of v5.4.4
+│  ├─ 404.html
+│  ├─ index.html
+│  ├─ ...
+├─ v5.4.3   <- the full build of v5.4.3
+│  ├─ 404.html
+│  ├─ index.html
+│  ├─ ...
+```
+
+Nesting previous versions in the latest version works the way we want it to.
+
+To deploy a new version:
+
+1. Copy all `docs/vN.N.N/` directories somewhere safe
+2. Rename `docs/` to `vN.N.N+1` (or whatever version it is)
+3. Add the docs for the new version in `docs/`
+   - Don't forget to update `_static/versions.json`!
+4. Move all `v.N.N.N` and `vN.N.N+1` back into `docs/`
```

Comment:
This could maybe use a little more clarification. 
1. Do you want to **_Copy_** (meaning the original stays where it is)?
so that in...
2. the `docs/` directory that we rename still has all the previous versions in it also?

3. There is no `docs/` directory after it was renamed in 2, so I don't know if we want to simply say Add the docs to that location. Probably obvious enough, but I don't think the language is precise.

4. I had to read this more than once. I get it, but maybe there is an easier way to explain?

I thought your tree outline above clearly demonstrates the desired outcome though, so I am able to reason through the intent here.

Reply:
Maybe these steps are overkill. There's more than one way to accomplish the task.  

In the end, all that matters is the directory structure, which is outlined.